lib/web/apiserver.go

```diff
@@ -825,10 +825,6 @@ func (h *Handler) bindDefaultEndpoints() {
 	h.POST("/webapi/users/password/token", h.WithAuth(h.createResetPasswordToken))
 	h.POST("/webapi/users/privilege/token", h.WithAuth(h.createPrivilegeTokenHandle))
 
-	// Issues SSH temp certificates based on 2FA access creds
-	// TODO(Joerger): DELETE IN v18.0.0, deprecated in favor of mfa login endpoints.
-	h.POST("/webapi/ssh/certs", h.WithUnauthenticatedLimiter(h.createSSHCert))
-
```

Comment:
Are the MFA login endpoints also usable with no MFA?

Reply:
Yes, we could have switched to this unified flow ages ago. I suppose it just adds one round trip.